src/content/docs/infrastructure/host-integrations/host-integrations-list/elasticsearch/elasticsearch-integration.mdx

```diff
@@ -0,0 +1,284 @@
+  - On-host integrations
+  - On-host integrations list
+metaDescription: "New Relic's Elasticsearch integration: what data it reports and how to configure it."
+redirects:
+  - /docs/integrations/host-integrations/host-integrations-list/elasticsearch-monitoring-integration
+  - /docs/elasticsearch-monitoring-integration
+  - /docs/elasticsearchonhost-integration-new-relic-infrastructure
+---
+
+import windows from './images/windows.png';
+
+import ecs from './images/ecs.png';
+
+import kubernetes from './images/kubernetes-k8.png';
+
+Our Elasticsearch [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your [Elasticsearch](https://www.elastic.co/) cluster to our platform, where you can see the health of your Elasticsearch environment. We collect metrics at the cluster, node, and index level so you can more easily find the source of any problems.
+
+To install the Elasticsearch monitoring integration, you must run through the following steps:
+
+1. [Configure the integration](#config).
+2. [Install and activate the integration](#install).
+3. [Find and use data](#find-and-use).
+4. Optionally, see [the advanced configuration settings](https://beta-docs.newrelic.com/docs/integrations/mySQL/mysql-advanced-config).
+
+## Compatibility and requirements [#req]
+
+### Elasticsearch versions [#elasticsearch-versions]
+
+Our integration is compatible with Elasticsearch 5.X through 7.X.
+
+### Supported operating systems [#supported-os]
+
+- Windows <ImageSizing width="32px" height="32px">![Windows](./images/windows.png 'Windows')</ImageSizing>
+- Linux <ImageSizing width="32px" height="32px">![Linux](./images/linux.png 'Linux')</ImageSizing>
+- MacOS <ImageSizing width="32px" height="32px">![MacOS](./images/macos.png 'MacOS')</ImageSizing>
+
+For a comprehensive list of OS versions, check the table of [compatible operating systems](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent/#operating-systems).
+
+### System requirements [#system-reqs]
+
+- A New Relic account. Don't have one? [Sign up for free!](https://newrelic.com/signup) No credit card required.
+- If Elasticsearch is not running on Kubernetes or Amazon ECS, you must [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic) on a host that's running Elasticsearch. Otherwise:
+  - If running on <ImageSizing width="32px" height="32px">![Kubernetes](./images/kubernetes-k8.png 'Kubernetes')</ImageSizing>Kubernetes, see [these requirements](/docs/monitor-service-running-kubernetes#requirements).
+  - If running on <ImageSizing width="32px" height="32px">![ECS](./images/ecs.png 'ECS')</ImageSizing>Amazon ECS, see [these requirements](/docs/integrations/host-integrations/host-integrations-list/monitor-services-running-amazon-ecs).
+
+## Configure the integration [#config]
```

Comment:
Shouldn't configuration go after install?

Reply:
This configuration needs to be done before installing the integration.